lib/CppInterOp/CppInterOp.cpp

```diff
@@ -3547,13 +3826,20 @@ bool InsertOrReplaceJitSymbol(compat::Interpreter& I,
   return false;
 }
 
-bool InsertOrReplaceJitSymbol(const char* linker_mangled_name,
-                              uint64_t address) {
-  return InsertOrReplaceJitSymbol(getInterp(), linker_mangled_name, address);
+bool InsertOrReplaceJitSymbol(const char* linker_mangled_name, uint64_t address,
+                              TInterp_t I /*=nullptr*/) {
+  auto* interp = static_cast<compat::Interpreter*>(I);
+  LOCK(getInterpInfo(interp));
+  if (interp)
+    return InsertOrReplaceJitSymbol(*interp, linker_mangled_name, address);
+  return InsertOrReplaceJitSymbol(getInterp(NULLPTR), linker_mangled_name,
+                                  address);
 }
 
 std::string ObjToString(const char* type, void* obj) {
-  return getInterp().toString(type, obj);
+  LOCK(getInterpInfo(NULLPTR)); // FIXME: not enough information to lock the
+                                // corrent interpreter
```

Comment:
```suggestion
                                // current interpreter
```
Please move the comment above the line.